src/sdk/models/smil-model.js

```diff
@@ -0,0 +1,760 @@
+//  LauncherOSX
```

Comment:
We don't need this file. This is all done [here]https://github.com/dariocravero/readium-js/blob/master/src/sdk/models/smil/index.js]. Just `require('sdk/models/smil');`.